.github/workflows/ci-benchmark.yaml

```diff
@@ -49,10 +49,15 @@ jobs:
       AIRFLOW__ASTRO_SDK__DATAFRAME_ALLOW_UNSAFE_STORAGE: True
       FORCE_COLOR: "true"
       ASTRO_PUBLISH_BENCHMARK_DATA: True
+      USE_GKE_GCLOUD_AUTH_PLUGIN: True
     steps:
       - uses: actions/checkout@v3
       - run: rm -f python-sdk/test-connections.yaml
       - run: ls -la
       - run: ( echo "cat <<EOF >python-sdk/test-connections.yaml"; cat .github/ci-test-connections.yaml; ) >python-sdk/test-connections.yaml && . python-sdk/test-connections.yaml
       - run: python -c 'import os; print(os.getenv("GOOGLE_APPLICATION_CREDENTIALS_JSON", "").strip())' > ${{ env.GOOGLE_APPLICATION_CREDENTIALS }}
+      - run: echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+      - run: curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+      - run: sudo apt update
```

Comment:
It would be nice if we could `name` the `- run:` jobs and split them into meaningful sections. WDYT?

Reply:
I split it into `Install google-cloud-sdk-auth-plugin` `Export environment variable` and `Run benchmark `.

Reply:
ok, we need this to run the gcloud/kubectl cmd which is a part of Run benchmark make file

Reply:
moved this in `Run benchmark`